mscore/menus.cpp

```diff
@@ -1308,6 +1308,65 @@ void MuseScore::setBasicPalette()
       }
 
 //---------------------------------------------------------
+//   genCreateMenu
+//---------------------------------------------------------
+
+QMenu* MuseScore::genCreateMenu(QWidget* parent)
```

Comment:
There is no function by this name in this file in the current master.  My guess is it existed a couple of months ago but was removed as part of @IsaacWeiss menu reorganization work, so somehow your code was not rebased correctly against that change.